File: iOS/DateMate/UI/SignIn/SignInUiState.swift
//
//  SignInUiState.swift
//  DateMate
//
//  Created by 홍희표 on 2025/11/18.
//

import Foundation

struct SignInUiState {
    var email: String = ""
    var emailError: String? = nil
    var password: String = ""
    var passwordError: String? = nil
    var isLoading: Bool = false
    var isSignInEnabled: Bool = false
}

File: Android/app/src/main/java/com/hhp227/datemate/ui/signin/SignInUiState.kt
package com.hhp227.datemate.ui.signin

data class SignInUiState(
    val email: String = "",
    val emailError: String? = null,
    val password: String = "",
    val passwordError: String? = null,
    val isLoading: Boolean = false,
    val isSignInEnabled: Boolean = false,
)